config/zend-sentry.global.php.dist

```diff
@@ -79,6 +79,11 @@ $settings = [
      */
     'ravenjs-version' => '3.27.0',
 
+    /**
+     * If you do not wish to use CDN to load raven-js you can specify path to raven-js to use
```

Comment:
Please correct the language by using the articles where they’re needed. The CDN, a path. And specify what kind of path you mean, relative to what. Add example.

Reply:
Done
I hope that does it since I've always had problems describing things :)